### PR TITLE
Update issue templates to remove discussions

### PR DESCRIPTION
**Pull request by ajcvickers**
Originally intended to just remove discussions, but this changed into add a template for asking a question, which itself changed into updating the templates.

Notes:
* Not sure if I'm dong this right
* Should the template text be in a comment?
* Should the template text be a template instead of instructions?
* Should we include real, runnable EF code as a starting point in the template?

Other suggestions?

### CODE REVIEW

**Comment by ajcvickers** [comment]
@dotnet/efteam Any thoughts on this? I would at least like to stop directing people to discussions ASAP.

File: .github/ISSUE_TEMPLATE/ask_a_question.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Finding DbContext classes in the project...
+Found DbContext 'BlogContext'.
+BlogContext
+```
+
+### Include provider and version information
+
+EF Core version:
+Database provider: (e.g. Microsoft.EntityFrameworkCore.SqlServer)
+Target framework: (e.g. .NET Core 3.0)
```
Comment (smitpatel) [inline .github/ISSUE_TEMPLATE/ask_a_question.md:50]: Tiny
```suggestion
Target framework: (e.g. .NET Core 5.0)
```

File: .github/ISSUE_TEMPLATE/bug_report_efcore.md [hunk 2]
```diff
@@ -4,29 +4,46 @@ about: Create a report about something that isn't working
 labels: customer-reported
 ---
 
-<!-- Describe what isn't working as expected -->
+### Include your code
 
+To fix any bug we mus first reproduce it. To make this possible, please attach a small, runnable project or post a small, runnable code listing that reproduces what you are seeing.
```
Comment (smitpatel) [inline .github/ISSUE_TEMPLATE/bug_report_efcore.md:9]:
```suggestion
To fix any bug we must first reproduce it. To make this possible, please attach a small, runnable project or post a small, runnable code listing that reproduces what you are seeing.
```

Comment (smitpatel) [reply .github/ISSUE_TEMPLATE/bug_report_efcore.md:9]: same in sqlite bug report

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,8 +1,5 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Got a question?
```
Comment (smitpatel) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Should we also add link to our docs here?
Check out documentation which may cover the answer of what you are looking for

**Review by smitpatel — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (ajcvickers) [reply .github/ISSUE_TEMPLATE/config.yml:3]: Good idea!

Comment (ajcvickers) [reply .github/ISSUE_TEMPLATE/config.yml:3]: @smitpatel Added some more text to the question and bug templates.

**Pull request merged**